README.md

```diff
@@ -1,3 +1,39 @@
-# coSMicQC
+<img height="200" src="https://raw.githubusercontent.com/WayScience/coSMicQC/main/logo/with-text-for-light-bg.png?raw=true">
 
-coSMicQC is a software tool for single-cell quality control of morphology datasets.
+# Single cell Morphology Quality Control
+
+[![Build Status](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml/badge.svg?branch=main)](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml?query=branch%3Amain)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+
```

Comment:
Suggestion to add Poetry and PyPI latest release.

```suggestion
[![Poetry](https://img.shields.io/endpoint?url=https://python-poetry.org/badge/v0.json)](https://python-poetry.org/)
![PyPI - Version](https://img.shields.io/pypi/v/cosmicqc)

```

Reply:
Awesome! I like those a lot! I have added them, thanks for the suggestion!